tests/queries/0_stateless/01600_detach_permanently.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../shell_config.sh
+
+## tests with real clickhouse restart would be a bit to heavy,
+## to ensure the table will not reappear back clickhose-local is enough.
```

Comment:
Yeah, i've reinvited that wheel in the 01601 test :) For some reason that idea didn't come to my mind at the beginining... But since the goal (creating the test) is archived, i would keep it as is.